src/modules/atommodel/RadixAtomModel.spec.ts

```diff
@@ -1,16 +1,20 @@
 import { expect } from 'chai'
 import 'mocha'
-import { RadixAtom } from '.'
+import { RadixAtom, RadixParticleGroup, RadixUniqueParticle, RadixAddress } from '.'
+import { RadixSpunParticle } from './particles/RadixSpunParticle';
 
 
 describe('Radix Atom model', () => {
 
     
     {
         const atom = new RadixAtom()
+        atom.particleGroups = [new RadixParticleGroup([RadixSpunParticle.up(
+            new RadixUniqueParticle(RadixAddress.generateNew(), 'test'),
+        )])]
         
         it(`should compute hid`, () => {
-            expect(atom.getHID.bind(atom)).to.not.throw()
```

Comment:
Can we maybe test more than just `not throw`, maybe at least that the text is not empty? Or better yet that it is a hex string of correct length?

Reply:
Might as well, this was initially introduced here and in all the particles for verifying the serialization pipeline mostly, but it's as good a spot as any for checking some more stuff about AIDs.